api/v1alpha1/project_types.go

```diff
@@ -88,9 +88,9 @@ type ProjectSpec struct {
 	// +optional
 	CategoryId int `json:"categoryId,omitempty"`
 
-	// The Custermer Access Status of a project
+	// The Access status of the project
```

Comment:
Add a little bit more details regarding what true or false in this case means. This will go in the field's definition in CRD